### DISCUSSION TITLE
Error when open virtual-display-driver-control

### DISCUSSION THREAD

**Discussion by Terminator8007** [MolotovCherry/virtual-display-rs#136] [Q&A]
I had an update of windows 11 and after that I was not able to enable the virtuals display, so I try to reinstal the software but after that when I open it I got this error messsage : panicked at virtual-display-driver-control\src\app.rs:76:14: called `Result::unwrap0` on an `Err` value: Error f code: HRESULT(Ox80070002), message: "Le fichier spécifié est introuvable." ).
![Capture d’écran 2024-05-21 160303](https://github.com/MolotovCherry/virtual-display-rs/assets/143729564/1ee8a8db-eab3-41a3-828f-d0cd0bae53d9)

**Comment by MolotovCherry** [comment]
Please see #49 for how to deal with this

**Reply by Terminator8007** [reply]
Tank you! Now everything work fine.
